### ISSUE TITLE
Support callback for send

### ISSUE THREAD

**Issue by 3rd-Eden** [honeybadger-io/honeybadger-node#5]
I want to know when stuff is send, without having to jumping through hoops and using the internal events. Just

```
hb.send(err, data, fn)
```

**Comment by joshuap** [comment]
Good idea. I'll add this to my todo list for the next release, although I would welcome a PR. :)

**Comment by joshuap** [comment]
I've addressed this in b786529bc7c1fc57af23417e08a8f300b4702ac0,which will be released with the next major version.

**Comment by joshuap** [comment]
Released in 1.0.0! Here's an example:

```javascript
Honeybadger.notify(err, data, function notifyCallback(err, notice) {
  if (err) {
    return console.error(err);
  }
  // If there was no error, log the notice:
  console.log(notice); // { id: 'uuid' }
});
```

(see [here](https://github.com/honeybadger-io/honeybadger-node#upgrading-from-0x) for upgrade instructions).
